src/core/thread/network_data_leader.cpp

```diff
@@ -479,6 +479,7 @@ ThreadError Leader::SetCommissioningData(const uint8_t *aValue, uint8_t aValueLe
         commissioningDataTlv->Init();
         commissioningDataTlv->SetLength(aValueLength);
         memcpy(commissioningDataTlv->GetValue(), aValue, aValueLength);
+        mLength += aValueLength;
```

Comment:
I thought `NetworkData::Insert()` increases `mLength` appropriately.  Why do we need this here?